src/Compilers/Core/Portable/Operations/Statement.cs

```diff
@@ -1,13 +1,20 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
+
 namespace Microsoft.CodeAnalysis.Semantics
 {
     internal partial class VariableDeclaration : IVariableDeclaration
     {
         public VariableDeclaration(ILocalSymbol variable, IOperation initialValue, SyntaxNode syntax) :
-            this(variable,
+            this(ImmutableArray.Create(variable), initialValue, syntax)
+        {
+        }
+
+        public VariableDeclaration(ImmutableArray<ILocalSymbol> variables, IOperation initialValue, SyntaxNode syntax) :
+            this(variables,
                 initialValue,
-                variable == null || (initialValue != null && initialValue.IsInvalid),
+                variables.IsDefault || (initialValue != null && initialValue.IsInvalid),
```

Comment:
Is there a scenario for a declaration with no variables? And if so, it seems like `variables` should be `ImmutableArray<ILocalSymbol>.Empty` rather than `default(ImmutableArray<ILocalSymbol>)`. (Otherwise, `GetDeclaredVariables()` will fail for instance.)